src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -615,6 +616,7 @@ public function formAction($id, Request $request)
             'is_shop_context' => $this->get('prestashop.adapter.shop.context')->isShopContext(),
             'editable' => $this->isGranted(PageVoter::UPDATE, 'ADMINPRODUCTS_'),
             'drawerModules' => $drawerModules,
+            'layoutTitle' => $this->trans('Product', 'Admin.Global'),
```

Comment:
minor issue, layoutTitle wasn't declared, and comes from AdminController (magically).